test/components/views/rooms/RoomTile-test.tsx

```diff
@@ -99,130 +108,199 @@ describe("RoomTile", () => {
+                        client.getSafeUserId(),
+                        client.getDeviceId()!,
+                        voiceBroadcastInfoEvent,
+                    );
+                    await act(async () => {
+                        room.currentState.setStateEvents([stopEvent]);
+                        await flushPromises();
+                    });
+                });
+
+                it("should not render the »Live« subtitle", () => {
+                    expect(screen.queryByText("Live")).not.toBeInTheDocument();
+                });
             });
         });
     });
 
-    describe("when a live voice broadcast starts", () => {
-        beforeEach(async () => {
-            await setUpVoiceBroadcast(VoiceBroadcastInfoState.Started);
+    describe("when message previews are enabled", () => {
+        beforeEach(() => {
+            showMessagePreview = true;
         });
 
-        it("should render the »Live« subtitle", () => {
-            expect(screen.queryByText("Live")).toBeInTheDocument();
+        it("should render a room without a message as expected", async () => {
+            renderRoomTile();
+            // flush promises here because the preview is created asynchronously
+            await flushPromises();
+            expect(renderResult.asFragment()).toMatchSnapshot();
         });
 
-        describe("and the broadcast stops", () => {
-            beforeEach(async () => {
-                const stopEvent = mkVoiceBroadcastInfoStateEvent(
-                    room.roomId,
-                    VoiceBroadcastInfoState.Stopped,
-                    client.getSafeUserId(),
-                    client.getDeviceId()!,
-                    voiceBroadcastInfoEvent,
-                );
-                await act(async () => {
-                    room.currentState.setStateEvents([stopEvent]);
-                    await flushPromises();
+        describe("and there is a message in the room", () => {
+            beforeEach(() => {
+                const message = mkMessage({
+                    event: true,
+                    room: room.roomId,
+                    msg: "test message",
+                    user: client.getSafeUserId(),
                 });
+
+                room.timeline.push(message);
+            });
+
+            it("should render as expected", async () => {
+                renderRoomTile();
+                expect(await screen.findByText("test message")).toBeInTheDocument();
+                expect(renderResult.asFragment()).toMatchSnapshot();
+            });
+        });
+
+        describe("and there is a message in a thread", () => {
+            beforeEach(() => {
+                const message = mkMessage({
+                    event: true,
+                    room: room.roomId,
+                    msg: "test thread reply",
+                    user: client.getSafeUserId(),
+                });
+
+                // Mock thread reply for tests.
```

Comment:
If anyone knows how to properly add a thread in a more „integration test“ fashioned way, please leave a comment.